lib/cli/kit/base_command.rb

```diff
@@ -0,0 +1,43 @@
+require 'cli/kit'
+
+module CLI
+  module Kit
+    class BaseCommand
+      def self.defined?
+        true
+      end
+
+      def self.statsd_increment(metric, **kwargs)
```

Comment:
I'll probably migrate `Statsd` over here too, but for now, these two are interface methods.